package.json

```diff
@@ -1,97 +1,107 @@
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org"
+  },
+  "keywords": [
+    "miragon",
+    "modeler",
+    "camunda",
+    "bpmnio",
+    "bpmn",
+    "dmn"
+  ],
+  "license": "Apache-2.0",
+  "files": [
+    "dist/",
+    "index.d.ts",
+    "index.js"
+  ],
+  "main": "index.js",
+  "directories": {
+    "lib": "dist/"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
     ],
-    "license": "Apache-2.0",
-    "files": [
-        "dist/",
-        "index.d.ts",
-        "index.js"
-    ],
-    "main": "index.js",
-    "directories": {
-        "lib": "dist/"
-    },
-    "browserslist": {
-        "production": [
-            ">0.2%",
-            "not dead",
-            "not op_mini all"
-        ],
-        "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
-        ]
-    },
-    "dependencies": {
-        "@material-ui/styles": "^4.11.4",
-        "@seznam/compose-react-refs": "^1.0.6",
-        "@uiw/react-monacoeditor": "^3.2.2",
-        "bpmn-js": "^8.7.0",
-        "bpmn-js-properties-panel": "^0.46.0",
-        "camunda-bpmn-moddle": "^5.1.2",
-        "camunda-dmn-moddle": "^1.1.0",
-        "clsx": "^2.0.0",
-        "deepmerge": "^4.2.2",
-        "diagram-js-origin": "^1.3.2",
-        "dmn-js": "^10.3.0",
-        "dmn-js-properties-panel": "^0.7.0",
-        "react-split-pane": "^0.1.92"
-    },
-    "devDependencies": {
-        "@rollup/plugin-commonjs": "^25.0.0",
-        "@rollup/plugin-node-resolve": "^15.0.1",
-        "@rollup/plugin-replace": "^5.0.2",
-        "@types/react": "^17.0.11",
-        "@typescript-eslint/eslint-plugin": "^4.27.0",
-        "@typescript-eslint/parser": "^4.27.0",
-        "babel-eslint": "^10.1.0",
-        "copyfiles": "^2.4.1",
-        "eslint": "^7.28.0",
-        "eslint-config-airbnb-typescript": "^14.0.2",
-        "eslint-config-react-app": "^7.0.1",
-        "eslint-plugin-flowtype": "^8.0.3",
-        "eslint-plugin-import": "^2.23.4",
-        "eslint-plugin-jsx-a11y": "^6.4.1",
-        "eslint-plugin-react": "^7.24.0",
-        "eslint-plugin-react-app": "^6.2.2",
-        "eslint-plugin-react-hooks": "^4.2.0",
-        "react": "^17.0.0",
-        "rollup": "^2.78.0",
-        "rollup-plugin-css-only": "^4.3.0",
-        "rollup-plugin-json": "^4.0.0",
-        "rollup-plugin-terser": "^7.0.2",
-        "rollup-plugin-typescript2": "^0.36.0",
-        "tslib": "^2.3.0",
-        "typedoc": "^0.25.1",
-        "typescript": "^5.2.2"
-    },
-    "peerDependencies": {
-        "react": "^17.0.0"
-    }
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "dependencies": {
+    "@babel/eslint-parser": "^7.24.8",
+    "@bpmn-io/properties-panel": "^3.22.3",
+    "@material-ui/styles": "^4.11.5",
```

Comment:
I [migrated](https://mui.com/material-ui/migration/migration-v4/) to `mui v5`. Furthermore instead of *JSS* I added [tss-react](https://mui.com/material-ui/migration/migrating-from-jss/#2-use-tss-react) which has a similiar API but uses *Emotion* instead.